src/services/hasura/hasura.service.ts

```
@@ -65,52 +65,133 @@ export class HasuraService {
+          case 'match':
+          case '=':
+            // Check if the filter value matches the first condition value
+            return cleanConditionValues[0] === cleanFilterValue || 
+              cleanConditionValues[0].replace(/[^a-zA-Z0-9]/g, '') === cleanFilterValue.replace(/[^a-zA-Z0-9]/g, '');
+
+          case 'lessthanequals':
+          case 'lte':
+          case 'lessthanorequal':
+          case '<=':
+            // Handle numeric comparison with first value
+            const filterNum = parseFloat(cleanFilterValue);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)